src/bin/pgcopydb/ld_transform.c

```diff
@@ -2061,7 +2061,7 @@ stream_write_insert(FILE *out, LogicalMessageInsert *insert)
 		/*
 		 * First, the PREPARE part.
 		 */
-		appendPQExpBuffer(buf, "INSERT INTO %s.%s ",
+		appendPQExpBuffer(buf, "INSERT INTO \"%s\".\"%s\" ",
```

Comment:
how about using `quote_identifier` from `"utils/builtins.h"`? @dimitri thoughts?

Reply:
If that's available to pgcopydb through libpq I think we should see about that, otherwise we can use [PQescapeIdentifier](https://www.postgresql.org/docs/current/libpq-exec.html#LIBPQ-PQESCAPEIDENTIFIER).

Reply:
Oh I think it isn't available through libpq and I have some wrong impression of using postgres internal libraries for our usage.
I think `PQescapeIdentifier` is our obvious choice then.